includes/class.llms.template.loader.php

```diff
@@ -93,8 +103,17 @@ public function maybe_redirect_to_sales_page() {
 			return;
 		}
 
+		$notices_to_print = LLMS()->session->get( 'llms_notices', array() );
```

Comment:
We can use `llms_notice_count()` instead.

Since we know we want to remove the reliance on the bundled session library using this function instead of checking the session data directly will make it easier to transition out in the future.

Reply:
https://developer.lifterlms.com/reference/functions/llms_notice_count/

Reply:
I'm sorry, I totally forgot about this helper!

Reply:
done in 3d1a8cc